fire/docstrings.py

```diff
@@ -336,9 +342,10 @@ def _as_arg_name_and_type(text):
     None otherwise.
   """
   tokens = text.split()
+  is_type = any(c in "[](){}" for c in text)
```

Comment:
What was the reason for this line?

Reply:
> I haven't tested this, but if we add : after the first word in descriptions (line2 or later), the _is_arg_name() function will be called and according to our current logic, anything that is just a word will be considered as an arg. I think we want a better way to identify an arg.

This was the initial reason for this. I just came up with this for the time being but i think that we need a better way to identify an arg name.